sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -537,9 +537,6 @@ export interface TextAnalyticsOperationOptions extends OperationOptions {
 // @public
 export type TextAnalyticsOperationStatus = "notStarted" | "running" | "succeeded" | "failed" | "rejected" | "cancelled" | "cancelling" | "partiallyCompleted" | "partiallySucceeded";
 
-// @public
-export type TextAnalyticsResult = TextAnalyticsSuccessResult | TextAnalyticsErrorResult;
```

Comment:
Unfortunately, this would be a breaking change. An end user may have written `import { TextAnalyticsResult } from "@azure/ai-text-analytics"`, and that would break.

Reply:
I understand that could be the case but ultimately this should not be a supported user scenario because this type in our client.

Reply:
@bterlson and @xirzec what do you think?

Reply:
I think that while technically it's changing public surface, it's only a type (so it's immaterial to JS customers), and if we never returned it or used it in a sample, it's pretty low risk. Perhaps a compromise could be doing a minor version bump? That way someone who is a stickler for patch updates won't be affected.

Reply:
Thanks @xirzec! I also talked to @bterlson privately and he said it is not a breaking change as long as there is no reason for the user to use it.